.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: ruby
```

Comment:
Based on the discussion in this PR, we should probably remove this `.travis.yml` file, no?